jupyterhub_singleuser_profiles/profiles.py

```diff
@@ -180,14 +180,16 @@ def setup_services(self, spawner, image, user):
     profile = self.get_merged_profile(image, user)
     if profile.get("services"):
       for key, service in profile.get("services").items():
```

Comment:
I know it is not part of the PR, but can you rename the `key` to `service_name` so that it is a bit more obvious?